docs/en/student-examples.md

```diff
@@ -0,0 +1,134 @@
+# Student Help Page
+
+This page describes how to interact with Ypovoli as a user.
+
+
+## Login/Logout
+
+**Login**
+
+- On the login page, there is a button with the text "Ugent login".
+
+![login button](../assets/login-button.png)
+- Click on this button. You will be redirected to the Ugent login page.
+- Select the desired account and follow the login steps. You will be redirected to the dashboard.
+
+**Logout**
+
+- Click on your name in the navigation bar.
+
+![logout button](../assets/en/logout-button.png)
+
+## Change Language
+
+- Click on the arrow next to EN at the top of the navigation bar and select the desired language.
+
+![language choice button](../assets/en/lang-change.png)
+
+## Change Mode
+
+::: danger Caution!
+- Only possible if the user has multiple roles like student, teacher, assistant, or admin.
+:::
+
+- Click on the arrow next to the currently selected role in the navigation bar at the top.
+- Choose the new role with which you want to view the page.
+
+![mode choice button](../assets/modus-change.png)
+
+## Enroll/leave in Course
```

Comment:
I believe "in" here was meant to be couple with enroll. So "Enroll in/Leave Course", I guess it would be.